go-controller/pkg/node/management-port_linux_test.go

```diff
@@ -454,8 +461,7 @@ func testManagementPortDPUHost(ctx *cli.Context, fexec *ovntest.FakeExec, testNS
 	err = testNS.Do(func(ns.NetNS) error {
 		defer GinkgoRecover()
 
-		mgmtPorts := NewManagementPorts(nodeName, nodeSubnetCIDRs)
-
+		mgmtPorts := NewManagementPorts(nodeName, nodeSubnetCIDRs, "pf0vf0", "")
```

Comment:
Inconsistency with a code above. Either pass values directly without intermediate variables for all test cases or assign values for this test case too.